github-electron/github-electron.d.ts

```diff
@@ -350,6 +350,15 @@ declare namespace Electron {
 		 */
 		isUnityRunning(): boolean;
 		/**
+		 * Returns a Boolean, true if Chrome's accessibility support is enabled, false otherwise.
```

Comment:
I would change it to "@returns If Chrome's accessibility support is enabled". Specifying the return type is not necessary, that's visible from the method signature.